libs/zgpu/build.zig

```diff
@@ -1,6 +1,7 @@
 const std = @import("std");
 const glfw = @import("../mach-glfw/build.zig");
 const gpu_dawn = @import("../mach-gpu-dawn/build.zig");
+const zpool = @import("../zpool/build.zig");
```

Comment:
Just trying to follow the pattern here and below, but I'm clearly missing something important.